package/gluon-ddhcpd/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-ddhcpd
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-ddhcpd
+  TITLE:=Distributed DHCP Daemon for Gluon
+  DEPENDS:=+gluon-core +ddhcpd
```

Comment:
If `gluon-ddhcpd` depends on `mmfd` that dependency is missing here.

Reply:
ddhcpd depends on a means to distribute multicast throughout the network. In babel networks this capability is provided by mmfd. in batman networks this is provided by the l2-property of batman. It could also be provided by bier or any other means. There is no way to model that currently and depending on mmfd is not the right thing to do for batman - as such a dependency was not placed.